docs/fundamentals/dashboard/standalone-for-nodejs.md

```diff
@@ -0,0 +1,210 @@
+>
+> - Start the .NET Aspire dashboard in standalone mode.
+> - Use the .NET Aspire dashboard with a Node.js app.
+
+## Prerequisites
+
+To complete this tutorial, you need the following:
+
+- [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/).
+  - You can use an alternative container runtime, but the commands in this article are for Docker.
+- [Node.js 18 or higher](https://nodejs.org/en/download/package-manager) locally installed.
+- A sample application.
+
+## Sample application
+
+This tutorial uses an Express.js application to demonstrate the integration. You can either create a new Express.js app or use an existing Node.js application.
+
+To create a new Express.js application:
+
+1. Create a new directory for your application:
+
+    ```console
+    mkdir aspire-nodejs-sample
+    cd aspire-nodejs-sample
+    ```
+
+1. Initialize a new Node.js project:
+
+    ```console
+    npm init -y
+    ```
+
+1. Install Express.js:
+
+    ```console
+    npm install express
+    ```
+
+1. Create a basic Express.js application by creating an `app.js` file:
+
+    ```javascript
+    const express = require('express');
+    const app = express();
+    const port = process.env.PORT || 3000;
+
+    app.get('/', (req, res) => {
+        console.log('Received request for home page');
+        res.json({
+            message: 'Hello from Node.js!',
+            timestamp: new Date().toISOString()
+        });
+    });
+
+    app.get('/api/weather', (req, res) => {
+        const weather = [
+            { city: 'Seattle', temperature: 72, condition: 'Cloudy' },
+            { city: 'Portland', temperature: 68, condition: 'Rainy' },
+            { city: 'San Francisco', temperature: 65, condition: 'Foggy' },
+            { city: 'Los Angeles', temperature: 78, condition: 'Sunny' }
+        ];
+        
+        console.log('Received request for weather data');
+        res.json(weather);
+    });
+
+    app.listen(port, () => {
+        console.log(`Server running at http://localhost:${port}`);
+    });
+    ```
+
+1. Update the `package.json` file to include a start script:
+
+    :::code language="json" source="~/docs/fundamentals/dashboard/snippets/standalone-for-nodejs/package.json":::
+
+1. Test the application by running:
+
+    ```console
+    npm start
+    ```
+
+1. Browse to the application at `http://localhost:3000` in a web browser to verify it's working.
+
+## Adding OpenTelemetry
+
+To use the .NET Aspire dashboard with your Node.js app, you need to install the OpenTelemetry SDK and exporter. The OpenTelemetry SDK provides the API for instrumenting your application, and the exporter sends telemetry data to the .NET Aspire dashboard.
+
+1. Install the OpenTelemetry SDK and related packages:
+
+    ```console
+    npm install @opentelemetry/api @opentelemetry/sdk-node @opentelemetry/exporter-trace-otlp-grpc @opentelemetry/exporter-metrics-otlp-grpc @opentelemetry/auto-instrumentations-node
+    ```
```

Comment:
This step can be omitted since we're having the user instead replace the entire _package.json_.